core/page.ml

```diff
@@ -87,6 +87,18 @@ module Make_RealPage (C : JS_PAGE_COMPILER) (G : JS_CODEGEN) = struct
                         ^ env
                         ^ head
                         ^ script_tag (String.concat "\n" defs)
+                        ^ "<script type=\"text/javascript\">
+                             'use strict';
+                             function _isRuntimeReady() {
+                                if (window._JSLIB === void 0 || window._JSLIB !== true) {
+                                   const msg = \"<h1>Startup error: Runtime dependency `jslib.js' is not loaded.</h1>\";
+                                   document.body.innerHTML = msg;
```

Comment:
Will this mean that the message is displayed twice if the page has a `<head>` and a `<body>`? If so, would it be worth just doing `document.innerHTML = <html><body>...</body></html>`?

Reply:
Actually, testing this, it's fine. Let's merge.

Reply:
No. It resets the head and body. The body displays the message in text, whilst the header sets the page title to be the message.